clang/test/SemaSYCL/intel-fpga-local.cpp

```diff
@@ -91,185 +91,187 @@ void foo1()
   //CHECK-NEXT: ConstantExpr
   //CHECK-NEXT: IntegerLiteral{{.*}}16{{$}}
   [[intelfpga::bank_bits(2,3), intelfpga::bankwidth(16)]]
-  unsigned int v_thirteen[64];
+  unsigned int bank_bits_width[64];
 
-  //CHECK: VarDecl{{.*}}v_fourteen
+  //CHECK: VarDecl{{.*}}doublepump_mlab
   //CHECK: IntelFPGADoublePumpAttr
   //CHECK: IntelFPGAMemoryAttr{{.*}}MLAB{{$}}
   [[intelfpga::doublepump]]
   [[intelfpga::memory("MLAB")]]
-  unsigned int v_fourteen[64];
+  unsigned int doublepump_mlab[64];
 
-  //CHECK: VarDecl{{.*}}v_fifteen
+  //CHECK: VarDecl{{.*}}mlab_doublepump
```

Comment:
I don't recall if there's a reason to do both `mlab_doublepump` and `doublepump_mlab`. Could you please bring it up/add it as a comment, or remove the extra case?

Reply:
Will remove duplicating case